install/kubernetes/cilium/templates/hubble/tls-cronjob/_job-spec.tpl

```diff
@@ -9,10 +9,18 @@ spec:
         {{- toYaml . | nindent 8 }}
         {{- end }}
     spec:
+      securityContext:
+        seccompProfile:
+          type: RuntimeDefault
```

Comment:
Shouldn't we also define the `appArmorProfile`?  cc @chancez

Reply:
Good point. @Sindvero can you do that? It will need to do a version check to make sure it's not enabled in versions that do not support it yet. You can look at 2136418dab6  for inspiration.

Reply:
Should we run with the "Unconfined" profile as well? @chancez

Reply:
@Sindvero I think we would want `RuntimeDefault` just like with seccomp.

Reply:
sounds good, just commited it